miio/integrations/vacuum/dreame/dreamevacuum_miot.py

```diff
@@ -210,17 +379,33 @@ class DreameVacuumMiot(MiotDevice):
             "Filter left level: {result.filter_left_time}\n"
             "Filter life level: {result.filter_life_level}\n"
             "Life brush main: {result.life_brush_main}\n"
-            "Life brush side: {result.life_brush_side}\n"
-            "Life sieve: {result.life_sieve}\n"
+            if hasattr(result, "life_brush_main")
+            else "" "Life brush side: {result.life_brush_side}\n"
```

Comment:
IMO, the container should always have this, and just return `None` if there is no value.